src/styles/_catalog.scss

```diff
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(2, 200px);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
It might not seem obvious, but in the requirements, it says `There should be two columns starting at 488px `so I guess that below 488px there should be 1 column layout.